zipline/gens/tradesimulation.py

```diff
@@ -215,8 +209,17 @@ def calculate_minute_capital_changes(dt):
 
             for dt, action in self.clock:
                 if action == BAR:
+                    cc = calculate_minute_capital_changes(dt)
```

Comment:
Rather than making this a separate step, could we just make `every_bar` return an iterator of packets to yield?  Same with `once_a_day` below.